services/QuillLMS/app/models/translation_prompts.rb

```diff
@@ -22,4 +22,15 @@ def self.feedback_strings_custom_prompt
   def self.feedback_strings_prompt(locale:)
     prompt_start(locale:) + feedback_strings_custom_prompt
   end
+
+  def self.format_locale(locale)
```

Comment:
Let's use a hash stored as a constant, which saves us memory from not having to do string allocations each time the method is called

```
  LOCALE_SCRIPTS = {
    'hi' => 'Hindi (Devangari script)',
    'ur' => 'Urdu (Nastaliq script)',
    'ko' => 'Korean (Hangul script)',
    'uk' => 'Ukrainian (Cyrillic script)',
    'ts' => 'Tagalog (Latin script)',
    'prs' => 'Dari (Perso-Arabic script)'
  }.freeze
  
  def self.format_locale(locale) = LOCALE_SCRIPTS.fetch(locale, locale)
```